src/player_input.py

```diff
@@ -10,54 +10,49 @@
 
 from __future__ import annotations
 from tokenize import String
+from typing import Optional
 import arcade
 from pyglet.input import Controller
 from pyglet.window.key import KeyStateHandler
 
 
 class PlayerInput:
-
-    # Left stick
-    x_axis: VirtualAxis
-    y_axis: VirtualAxis
-
-    # Right stick
-    rx_axis: VirtualAxis
-    ry_axis: VirtualAxis
-
-    # Buttons
-    accelerate_button: VirtualButton
-    brake_button: VirtualButton
-    primary_fire_button: VirtualButton
-    secondary_fire_button: VirtualButton
-
```

Comment:
Opted to move these all into `__init__`.  I go back and forth on preference here, but putting them inside `__init__` is the more foolproof option since they can be assigned values at the same time.  Assigning values *outside* of `__init__` creates class attributes, not instance attributes.  Sometimes safe, sometimes not.